{{cookiecutter.repo_root}}/SHARED-CI.md

```diff
@@ -64,6 +64,25 @@ task                                    👈 all tasks go here
 
 ## ☑️ CI workflows
 
+### Kustomize Build
+
+- script: [`hack/build-manifests.sh`](hack/build-manifests.sh)
+  - Generates task manifest YAML files from Kustomize definitions (kustomize.yaml, patch.yaml)
+- script: [`hack/verify-manifests.sh`](hack/verify-manifests.sh)
+  - Verifies if all task manifests are up to date (no rebuild required).
+- workflow: [`.github/workflows/check-kustomize-build.yaml`](.github/workflows/check-kustomize-build.yaml)
+  - Checks if all task manifests are up to date (no rebuild required).
+
+With Kustomize, Task manifests are generated and kept consistent across the
+repository by composing base definitions (kustomize.yaml) with patches (patch.yaml).
+This ensures that all Task YAML manifests are reproducible and remain in sync 
+with their source definitions.
+
+When authoring or modifying a Task, contributors should update the corresponding
+Kustomize files and regenerate the manifests rather than editing the YAML directly.
+Use [`hack/build-manifests.sh`](hack/build-manifests.sh) to regenerate the manifests, and
+[`hack/verify-manifests.sh`](hack/verify-manifests.sh) to check that they are up to date.
```

Comment:
I think we don't need to document the verify script. It doesn't do much, and the developer is better off just running `git diff`

We could even delete the script and inline the check in the GH workflow. I don't think it has much value as a standalone script.

Reply:
removed the mention